responders/src/UI/embc-responder/src/app/shared/components/dialog-components/registrant-link-dialog/registrant-link-dialog.component.html

```diff
@@ -41,14 +41,16 @@
         <p class="p-margin">BC Services Card Online Authenticated:</p>
       </div>
       <div class="col-md-5 bold">
-        <p class="p-margin">{{ profile?.authenticatedUser ? 'Yes' : 'No' }}</p>
+        <p class="p-margin">
+          {{ profile?.status === 'Verified' ? 'Yes' : 'No' }}
+        </p>
       </div>
     </div>
     <div class="row">
       <div class="col-md-6"><p class="p-margin">Verification Status:</p></div>
       <div class="col-md-6 bold">
         <p class="p-margin">
-          {{ profile?.verifiedUser ? 'Verified' : 'Unverified' }}
+          {{ profile?.isAuthenticated ? 'Verified' : 'Unverified' }}
```

Comment:
isAuthenticated is not the same as isVerified. Verified refers to gov-id while authenticated referes to bcsc authentication